src/App.vue

```diff
@@ -1,63 +1,86 @@
 <script>
 export default {
   name: 'App',
-  methods: {
-    onNav (name) {
-      this.$router.push({path: `/${name}`})
+  data () {
+    return {
+      dialogFormVisible: false,
+      form: {}
     }
   }
 }
 </script>
 
 <template>
-  <main>
-    <at-menu mode="horizontal" @on-select="onNav">
+  <div class="app">
+
+    <el-dialog title="create new game" :visible.sync="dialogFormVisible">
+      <el-form :model="form">
+        <el-form-item label="display name">
+          <el-input v-model="form.name" auto-complete="off"></el-input>
+        </el-form-item>
+        <el-form-item label="slug">
+          <el-input v-model="form.slug" auto-complete="off"></el-input>
+        </el-form-item>
+      </el-form>
+      <span slot="footer" class="dialog-footer">
+        <el-button @click="dialogFormVisible = false">Cancel</el-button>
+        <el-button type="primary" @click="dialogFormVisible = false">Confirm</el-button>
+      </span>
+    </el-dialog>
+
+    <el-menu mode="horizontal" :router="true">
```

Comment:
Let's use the prepackaged way to do layouts in Element UI, it's basically made for the look of our application: [Container](http://element.eleme.io/#/en-US/component/container)